students/marc_charbo/assignment_1/generator_solution.py

```diff
@@ -0,0 +1,33 @@
+def doubler(max):
```

Comment:
So these functions should be testable via the `test_generator.py`.  You want to give it another shot?

Reply:
added unittest lib and class to test